src/OrchardCore/OrchardCore.HealthChecks.Abstractions/HealthChecksOptions.cs

```diff
@@ -9,5 +9,10 @@ public class HealthChecksOptions
         /// Gets or sets the health check URL. Default to "/health/live".
         /// </summary>
         public string Url { get; set; } = "/health/live";
+
+        /// <summary>
+        /// Whether to show the details for the checks dependency or not. Default to <c>false</c>.
```

Comment:
Explain in the comment what "details" are.

Reply:
I just checked both cases with & without show details everything works as expected